src/destreamer.ts

```diff
@@ -286,12 +339,22 @@ async function main(): Promise<void> {
         [videoGUIDs, outDirs] =  await parseInputFile(argv.inputFile!, argv.outputDirectory, session);
     }
 
-    logger.verbose('List of GUIDs and corresponding output directory \n' +
+    logger.verbose('List of videos and corresponding output directory \n' +
         videoGUIDs.map((guid: string, i: number) =>
             `\thttps://web.microsoftstream.com/video/${guid} => ${outDirs[i]} \n`).join(''));
 
 
-    downloadVideo(videoGUIDs, outDirs, session);
+    /* FIXME: [FATAL] we have 4 lingering socket connections and
+    I can't figure them out.
+    To see them use debug mode and after execution (I suggested using
+    --simulate) use 'process._getActiveHandles();' and
+    'process._getActiveRequests();' in the debug console to see lingering
+    Handles (where you can find the sockets) or Requests */
+    await downloadVideo(videoGUIDs, outDirs, session);
+
+    // workaround for issue above
```

Comment:
We can worry about cleaning up later. 4 dirty sockets is NOTHING in the big scheme of things.

_65531 sockets should be enough for everyone_ (Bill Gates if he would have reviewed this back in the 80s)

Reply:
the funny thing is that they don't connect to anything that I can see during execution :/